FirebaseGoogleAuthUI/Sources/FUIGoogleAuth.m

```diff
@@ -102,6 +100,13 @@ - (instancetype)initWithScopes:(NSArray *)scopes {
 }
 #pragma clang diagnostic pop
 
+- (GIDSignIn *)googleSignIn {
+  return [GIDSignIn sharedInstance];
```

Comment:
Should this be `GIDSignIn.sharedInstance`?

From https://developers.google.com/identity/sign-in/ios/quick-migration-guide: "Update all [GIDSignIn sharedInstance] calls to GIDSignIn.sharedInstance property accesses."

Reply:
Done.